packages/gestalt/src/icons/globe-checked.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" ><path fill="#B5B5B5" d="M6.783,9.846 L3.133,9.846 C3.51,6.999 5.42,4.634 8.007,3.609 C7.29,5.263 6.877,7.519 6.783,9.846 M3.133,11.903 L6.783,11.903 C6.877,14.231 7.29,16.486 8.007,18.14 C5.42,17.115 3.51,14.75 3.133,11.903 M10.879,3.057 C11.432,3.107 12.727,5.531 12.909,9.846 L8.842,9.846 C9.031,5.465 10.386,3.091 10.879,3.057 M9.537,16.273 C9.202,15.17 8.92,13.72 8.842,11.903 L11.514,11.903 C12.437,10.91 13.622,10.168 14.963,9.789 C14.865,7.482 14.455,5.25 13.746,3.61 C16.278,4.614 18.15,6.904 18.582,9.671 C19.349,9.836 20.071,10.122 20.732,10.503 C20.534,5.233 16.199,1.004 10.883,1 L10.879,1 L10.875,1 C5.43,1 1,5.429 1,10.875 C1,16.194 5.231,20.533 10.504,20.731 C9.869,19.63 9.5,18.359 9.5,17 C9.5,16.755 9.514,16.513 9.537,16.273 M20.287,15.775 L16.174,20 L13.713,17.472 C13.429,17.18 13.429,16.707 13.713,16.415 C13.998,16.123 14.458,16.123 14.742,16.415 L16.174,17.886 L19.258,14.719 C19.542,14.427 20.003,14.427 20.287,14.719 C20.571,15.01 20.571,15.483 20.287,15.775 M17,11 C13.687,11 11,13.686 11,17 C11,20.314 13.687,23 17,23 C20.314,23 23,20.314 23,17 C23,13.686 20.314,11 17,11"/></svg>
```

Comment:
@BigTiannn Could you verify that this went through PDS (@jaymarsh) and is the the PDS `Web spec` directory? It doesn't look optimized:
![image](https://user-images.githubusercontent.com/127199/41922519-1d756bfc-791a-11e8-93b9-f3ba22d7fead.png)

Reply:
@christianvuerings This is the svg that @jaymarsh gave to me. I can double check with him.